src/AutoRest.CSharp/Common/AutoRest/Communication/StandaloneGeneratorRunner.cs

```diff
@@ -255,6 +255,7 @@ internal static void LoadConfiguration(string? projectPath, string outputPath, s
                 ReadOption(root, Configuration.Options.SingleTopLevelClient),
                 ReadOption(root, Configuration.Options.SkipSerializationFormatXml),
                 ReadOption(root, Configuration.Options.DisablePaginationTopRenaming),
+                ReadOption(root, Configuration.Options.ArrayToPage),
```

Comment:
OK I get it since this configuration is only used in cadl projects, and for cadl projects the configurations are written by the emitter, we do not need to write the configuration in the generator.

But should we align? or add some comments to explain?